tf2onnx/tflite/TensorType.py

```diff
@@ -21,4 +21,5 @@ class TensorType(object):
     RESOURCE = 13
     VARIANT = 14
     UINT32 = 15
+    UINT16 = 16
```

Comment:
It looks like we need add the `TFLiteTensorType` in [`tf2onnx/tflite_utils.py`](https://github.com/onnx/tensorflow-onnx/blob/3e95ee7083af2a70a5ef4c09c1d4448a32467a69/tf2onnx/tflite_utils.py#L34) at the same time.